org.sabnzbd.sabnzbd.yaml

```diff
@@ -63,47 +121,53 @@ modules:
     build-commands:
       - pip3 install --verbose --exists-action=i --no-index --find-links="file://${PWD}" --prefix=${FLATPAK_DEST} "cryptography" --no-build-isolation
     sources:
-      - cargo-sources.json
+      - cargo-sources-cryptography.json
       - type: file
-        url: https://files.pythonhosted.org/packages/78/63/66c03eb51f0d241862083deb3f17ab5fce08cf6b347db7887bcb4d1a194e/cryptography-42.0.7.tar.gz
-        sha256: ecbfbc00bf55888edda9868a4cf927205de8499e7fabe6c050322298382953f2
+        # Remember to update the associated Cargo-*.lock and regenerate cargo-sources-*.json on every version bump!
+        url: https://files.pythonhosted.org/packages/a7/35/c495bffc2056f2dadb32434f1feedd79abde2a7f8363e1974afa9c33c7e2/cryptography-45.0.7.tar.gz
+        sha256: 4b1654dfc64ea479c242508eb8c724044f1e964a47d1d1cacc5132292d851971
         x-checker-data:
           type: pypi
           name: cryptography
+
   - name: p7zip
     no-autogen: true
     make-args:
       - 7z
     sources:
-      - type: archive
-        url: https://github.com/p7zip-project/p7zip/archive/v17.05/p7zip-v17.05.tar.gz
-        sha256: d2788f892571058c08d27095c22154579dfefb807ebe357d145ab2ddddefb1a6
+      - type: git
+        url: https://github.com/p7zip-project/p7zip
+        tag: v17.06
       - type: shell
         commands:
           - sed -i 's|/usr/local|/app|g' makefile.common
+
   - name: unrar
     no-autogen: true
     make-install-args:
       - DESTDIR=$(FLATPAK_DEST)
     sources:
       - type: archive
-        url: https://www.rarlab.com/rar/unrarsrc-7.0.9.tar.gz
-        sha256: 505c13f9e4c54c01546f2e29b2fcc2d7fabc856a060b81e5cdfe6012a9198326
+        # Find the "UnRAR source" link at https://www.rarlab.com/rar_add.htm
+        url: https://www.rarlab.com/rar/unrarsrc-7.2.1.tar.gz
+        sha256: 3fe3b4d710da45521625353dc2e023dad48c010f02a93302756e1061a8f3ae8e
       - type: patch
+        # Patch origin is Debian, see https://github.com/debian-calibre/unrar-nonfree/tree/master/debian/patches
         path: patches/0001-fix-buildflags.patch
-        sha256: ecd54158acaf7562585d465c8f613aa35130731acf270e4224fd9bd8f00e25b4
```

Comment:
Are you sure a checksum isn't needed?

Reply:
Had to take it out, it caused the test build here to fail with a linter error (see initial run on this PR)